timescaledb/how-to-guides/update-timescaledb/update-timescaledb.md

```diff
@@ -0,0 +1,68 @@
+# Updating major versions of TimescaleDB [](update)
+
+Use these instructions to update TimescaleDB within the same major release
+version (for example, from TimescaleDB 2.1 to 2.2, or from 1.7 to 1.7.4). If you need to upgrade between
+TimescaleDB 1.x and 2.x, see our [separate upgrade document][update-tsdb-2] 
+for detailed instructions.
+
+### TimescaleDB release compatibility
+
+TimescaleDB currently supports the following PostgreSQL releases. If you are not currently running
+a compatible release, please upgrade before updating TimescaleDB.
+
+ TimescaleDB Release |   Supported PostgreSQL Release
+ --------------------|-------------------------------
+ 1.7                 | 9.6, 10, 11, 12
+ 2.0                 | 11, 12
+ 2.1+                | 11, 12, 13
+
+If you need to upgrade PostgreSQL first, see [our documentation][upgrade-pg].
```

Comment:
Explicitly note that if you want to upgrade both TimescaleDB & major PostreSQL versions, this should be done as two separate processes.  E.g., do a full TimescaleDB upgrade, then to PG upgrade.  Or, do full PG upgrade then TimescaleDB upgrade.  Don't try to combine them.